.github/workflows/tests.yml

```diff
@@ -184,8 +168,8 @@ jobs:
           python-version: 3.7
       - name: Build package
         run: |
-          python -m pip install -U pip setuptools wheel
-          python setup.py sdist bdist_wheel
+          python -m pip install -U pip build
+          python -m build
```

Comment:
Since we are using stb now, we should be using `stb package`, right?
https://sphinx-theme-builder.readthedocs.io/en/latest/tutorial/#packaging-the-theme

I guess this is a more general question... do we want to use stb commands here or the underlying machinery?

Reply:
This is more "correct" since it's portable across whatever build-backend is used here.

Plus, at the cost of sharing how the sausage is made, `stb package` is... uhm... a very dumb and thin wrapper around this command anyway. :)

https://github.com/pradyunsg/sphinx-theme-builder/blob/main/src/sphinx_theme_builder/_internal/cli/package.py

Reply:
That is the answer I needed, thanks for the feedback @pradyunsg!